pkg/client/cli/telepresence_test.go

```diff
@@ -1391,17 +1441,22 @@ func (ts *telepresenceSuite) kubectlOut(ctx context.Context, args ...string) (st
 func (ts *telepresenceSuite) publishManager() error {
 	ctx := dlog.NewTestContext(ts.T(), true)
 	cmd := dexec.CommandContext(ctx, "make", "push-image")
+	if goRuntime.GOOS == "windows" {
+		cmd = dexec.CommandContext(ctx, "winmake.bat", "push-image")
+	}
 
 	// Go sets a lot of variables that we don't want to pass on to the ko executable. If we do,
 	// then it builds for the platform indicated by those variables.
 	cmd.Env = []string{
 		"TELEPRESENCE_VERSION=" + ts.testVersion,
 		"TELEPRESENCE_REGISTRY=" + dtest.DockerRegistry(ctx),
 	}
-	includeEnv := []string{"HOME=", "PATH=", "LOGNAME=", "TMPDIR=", "MAKELEVEL="}
+	includeEnv := []string{"HOME=", "PATH=", "Path=", "LOGNAME=", "TMPDIR=", "MAKELEVEL="}
 	for _, env := range os.Environ() {
+		dlog.Infof(ctx, "I found variable %s", env)
```

Comment:
Not OK! This will leak CI secrets if the test fails!

Reply:
OMG this is very stupid on my part; I believe Circle should [mask](https://circleci.com/docs/2.0/env-vars/#secrets-masking) the secrets, and I haven't seen them be printed, but I'll definitely remove this and go check that it hasn't  been printed anywhere